.github/scripts/issue_tracker.py

```diff
@@ -3,45 +3,46 @@
 # TODO: The git blame is wrong. It seems to take the last commit instead.
```

Comment:
can we restore this file? looks like only cosmetic changes and i don't care about python formatting